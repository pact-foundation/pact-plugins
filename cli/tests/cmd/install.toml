bin.name = "pact-plugin"
args = "install --help"
