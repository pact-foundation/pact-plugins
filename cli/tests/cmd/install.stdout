Install a plugin

A plugin can be either installed from a URL, or for a known plugin, by name (and optionally version).

Usage: pact-plugin install [OPTIONS] <SOURCE>

Arguments:
  <SOURCE>
          Where to fetch the plugin files from. This should be a URL or the name of a known plugin

Options:
  -t, --source-type <SOURCE_TYPE>
          The type of source to fetch the plugin files from. Will default to Github releases.
          
          Valid values: github

  -y, --yes
          Automatically answer Yes for all prompts

  -s, --skip-if-installed
          Skip installing the plugin if the same version is already installed

  -v, --version <VERSION>
          The version to install. This is only used for known plugins

      --skip-load
          Skip auto-loading of plugin
          
          [env: PACT_PLUGIN_CLI_SKIP_LOAD=]

  -h, --help
          Print help (see a summary with '-h')
